hamilton_launch_server/livestream/ffserver.conf

```diff
@@ -0,0 +1,138 @@
+# Port on which the server is listening. You must select a different
+# port from your standard HTTP web server if it is running on the same
+# computer.
+HTTPPort 8090
+
+# Address on which the server is bound. Only useful if you have
+# several network interfaces.
+#HTTPBindAddress 0.0.0.0
+
+# Number of simultaneous HTTP connections that can be handled. It has
+# to be defined *before* the MaxClients parameter, since it defines the
+# MaxClients maximum limit.
+MaxHTTPConnections 2000
+
+# Number of simultaneous requests that can be handled. Since FFServer
+# is very fast, it is more likely that you will want to leave this high
+# and use MaxBandwidth, below.
+MaxClients 1000
+
+# This the maximum amount of kbit/sec that you are prepared to
+# consume when streaming to clients.
+MaxBandwidth 50000
+
+# Access log file (uses standard Apache log file format)
+# '-' is the standard output.
+CustomLog -
+
+##################################################################
+# Definition of the live feeds. Each live feed contains one video
+# and/or audio sequence coming from an ffmpeg encoder or another
+# ffserver. This sequence may be encoded simultaneously with several
+# codecs at several resolutions.
+
+<Feed feed1.ffm>
+
+# You must use 'ffmpeg' to send a live feed to ffserver. In this
+# example, you can type:
+#
+# ffmpeg http://localhost:8090/feed1.ffm
+
+# ffserver can also do time shifting. It means that it can stream any
+# previously recorded live stream. The request should contain:
+# "http://xxxx?date=[YYYY-MM-DDT][[HH:]MM:]SS[.m...]".You must specify
+# a path where the feed is stored on disk. You also specify the
+# maximum size of the feed, where zero means unlimited. Default:
+# File=/tmp/feed_name.ffm FileMaxSize=5M
+File feed1.ffm
+FileMaxSize 100M
+
+# You could specify
+# ReadOnlyFile /saved/specialvideo.ffm
+# This marks the file as readonly and it will not be deleted or updated.
+
+# Specify launch in order to start ffmpeg automatically.
+# First ffmpeg must be defined with an appropriate path if needed,
+# after that options can follow, but avoid adding the http:// field
+#Launch ffmpeg
+
+# Only allow connections from localhost to the feed.
+#ACL allow 127.0.0.1
+</Feed>
+
+<Feed feed2.ffm>
+File feed2.ffm
+FileMaxSize 100M
+</Feed>
+
+<Feed feed3.ffm>
+File feed3.ffm
+FileMaxSize 100M
+</Feed>
+
+
+# Multipart JPEG
+
+<Stream test1.mjpg>
+Feed feed1.ffm
+Format mpjpeg
+VideoFrameRate 60 
+VideoIntraOnly
+NoAudio
+Strict -1
+VideoBufferSize 80000
+VideoBitRate 4000
+VideoSize 640x480
+VideoQMin 1
+VideoQMax 5
+</Stream>
+
+<Stream test2.mjpg>
+Feed feed2.ffm
+Format mpjpeg
+VideoFrameRate 60 
+VideoIntraOnly
+NoAudio
+Strict -1
+VideoBufferSize 80000
+VideoBitRate 4000
+VideoSize 640x480
+VideoQMin 1
+VideoQMax 5
+</Stream>
+
+<Stream test3.mjpg>
+Feed feed3.ffm
+Format mpjpeg
+VideoFrameRate 60 
+VideoIntraOnly
+NoAudio
+Strict -1
+VideoBufferSize 80000
+VideoBitRate 4000
+VideoSize 640x480
+VideoQMin 1
+VideoQMax 5
```

Comment:
What are these values? `VideoQMin` and `VideoQMax`

Reply:
https://stackoverflow.com/questions/18563764/why-low-qmax-value-improve-video-quality?utm_medium=organic&utm_source=google_rich_qa&utm_campaign=google_rich_qa

I guess we'll have to play with these values. I just took whatever was default or what I had found earlier. 

_"values of qmin lower than 16 and qmax 26 are visibly 'very good' lowering qmin below 16 costs extra space 'without adding visible' quality. "_

I might try doing something like this^

Reply:
Actually, I tried changing it to that, and it looks a bit worse. I'll just leave it at what it is right now, and if the computer you are bringing seems to struggle, I guess play with the values + the resolutions